README.md

```diff
@@ -1,6 +1,6 @@
-# Neuro-San Data-Driven Agents
+# Neuro SAN Data-Driven Agents
```

Comment:
Consistently using the `Neuro SAN` spelling when not referring to the `neuro-san` GitHub repo